src/components/Display/Display.spec.js

```diff
@@ -2,11 +2,19 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import Display from './Display';
 
-describe('Calculator', () => {
-    let wrapper;
-    beforeEach(() => wrapper = shallow(<Display displayValue={'0'}/>))
+describe('Display', () => {
+    let subject,
+        props;
```

Comment:
```suggestion
        let subject;
        let props;
```
This is valid JS, however our eslint configuration will not allow this on EXH